.github/workflows/e2e-tests.yaml

```diff
@@ -0,0 +1,39 @@
+name: e2e-tests
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.10"]
+
+    steps:
+      - uses: actions/checkout@master
+        with:
+          repository: MagnivOrg/magniv-repo-for-testing
```

Comment:
Will need to put more stuff in this repo, but fine for this commit ✅

Reply:
yes agreed, figured maybe we can do that via PRs to the other repo and run this same test there too to make sure we don't break this repo

Reply:
Added this here if you can also take a look https://github.com/MagnivOrg/magniv-repo-for-testing/pull/1